src/test/java/core/basesyntax/FruitTransactionTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.FruitTransaction;
+import org.junit.jupiter.api.Test;
+
+public class FruitTransactionTest {
+
+    @Test
+    public void constructor_validParameters_createsFruitTransaction() {
+        FruitTransaction transaction
+                = new FruitTransaction(FruitTransaction.Operation.SUPPLY,
+                "apple", 10);
+
+        assertEquals(FruitTransaction.Operation.SUPPLY, transaction.getOperation());
+        assertEquals("apple", transaction.getFruit());
+        assertEquals(10, transaction.getQuantity());
+    }
+
+    @Test
+    public void getByCode_validCode_returnsCorrectOperation() {

```

Comment:
```suggestion
    public void getByCode_validCode_ok() {
```
use the same test naming style